modules/cert-manager-configure-cloud-credentials-aws-non-sts.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-aws.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-aws-non-sts"]
+= Configuring the {cert-manager-operator} to use prepared cloud credentials
+
+You can configure the {cert-manager-operator} to use prepared cloud credentials on AWS Security Token Service cluster by using the `ccoctl` binary.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+
+.Procedure
+
+. The {cert-manager-operator} creates the `CredentialsRequest` object in the `openshift-cloud-credential-operator` namespace. Get the role by running the following command:
++
+[source,terminal]
+----
+$ oc get credentialsrequest -n openshift-cloud-credential-operator -o yaml
+----
++
+.Example output
+[source,terminal]
+----
+arn:aws:iam::<AWS-account-id>:role/abcd-cert-manager
+----
+
+. Annotate the service account by running the following command:
++
+[source,terminal]
+----
+$ oc annotate sa/cert-manager -n cert-manager eks.amazonaws.com/role-arn="arn:aws:iam::<AWS-account-id>:role/abcd-cert-manager"
```

Comment:
You can skip this annotate step in non-sts AWS, as for most non-STS clusters they won't have the pod identity webhook functional (which is responsible for injecting the credentials into the pod base upon annotations in the service account).
xref: https://redhat-internal.slack.com/archives/C04TMSTHUHK/p1683663618177239?thread_ts=1683658420.220019&cid=C04TMSTHUHK